src/libguac/guacamole/protocol.h

```diff
@@ -38,6 +38,9 @@
 #include <cairo/cairo.h>
 #include <stdarg.h>
 
+/* The protocol version */
```

Comment:
This public API documentation for a macro like this needs to be more specific.

Reply:
Will beef it up a little.

Reply:
Hopefully the comment is adequate and accurate, now.